### DISCUSSION TITLE
Add additional music streaming links

### DISCUSSION THREAD

**Discussion by somaamos** [JulianPrieber/littlelink-custom#506] [Predefined Links]
A few additional music streaming links which are currently missing I can suggest:

Majors (most important):
Tidal
Pandora
iHeart (aka iHeartRadio)
Amazon-music (Amazon exists)

Global (nice to have):
Claro Musica
Anghami
Qobuz
Joox
NetEase
Saavn
Tencent
Yandex-music
Boomplay
Adaptr
Flo

**Comment by Lucyy-cloud** [comment]
[ViWizard Amazon Music Music Converter](https://www.viwizard.com/amazon-music-converter/), a professional tool for Amazon music streaming, is a perfect Amazon music downloader without any defects. It can help you download songs from Amazon Music Prime.
